src/wasmcloud.rs

```diff
@@ -1,20 +1,25 @@
-use actix_rt::System;
-use wascc_host::HostBuilder;
+use wasmcloud_host::{HostBuilder, Result};
 
 #[macro_use]
 extern crate log;
 
 #[actix_rt::main]
-async fn main() {
+async fn main() -> Result<()> {
     let _ = env_logger::Builder::from_env(env_logger::Env::default().filter_or(
         env_logger::DEFAULT_FILTER_ENV,
-        "wasccd=info,wascc_host=info",
+        "wasccd=info,wasmcloud_host=info",
```

Comment:
Will this log level set still be called `wasccd`, or will it be `wasmcloud`?

Reply:
should be wasmcloud